src/App/DocumentObjectPyImp.cpp

```diff
@@ -712,7 +718,7 @@ PyObject*  DocumentObjectPy::getPathsByOutList(PyObject *args)
     }
 }
 
-PyObject *DocumentObjectPy::getCustomAttributes(const char* attr) const
+PyObject *DocumentObjectPy::getCustomAttributes(const char* attr)

```

Comment:
You cannot simply remove the const here. The change causes a build failure.

Reply:
Cool. It did build here before I lodged, but not there! But yes, will reverse. And be more careful.